### DISCUSSION TITLE
Noticed in multitenant env

### DISCUSSION THREAD

**Discussion by ausangshukla** [excid3/noticed#428] [General]
I have a multitenant app. In the version < 2, I use to be able to store the tenant id in the notifications table and retrieve all notifications sent by the tenant.

How should I approach this in version > 2.
Note Im using mysql as the DB, rails 7.1, and noticed 2.2

Thanks

**Comment by excid3** [comment]
Add the tenant_id to the models and use a concern to add the `belongs_to` to them in a `config.to_prepare` block. That's what we do in Jumpstart Pro apps.

**Reply by excid3** [reply]
```ruby
module NotificationExtension
  extend ActiveSupport::Concern

  included do
    belongs_to :account
  end
end

Rails.configuration.to_prepare do
  Noticed::Event.include EventExtension
end
```
